ansible/configs/satellite-v64-prod/env_vars.yml

```diff
@@ -184,7 +183,6 @@ capsule_instances:
     count: "{{capsule_instance_count}}"
     security_groups:
       - CapsuleSG
-      - DefaultSG
```

Comment:
Why remove the defaultSG ?

Reply:
When deploying to OSP, a duplicate rule configuration causes the deployment to fail - whereas AWS copes with this just fine. The long-term solution is to deduplicate the security group declarations, but as capsules aren't even used in this configuration at present then removing DefaultSG here allows the Satellite deployment to complete in both OSP and AWS.

Reply:
OK if it works with both, all good